### DISCUSSION TITLE
How to set my password to prevent others from connect Milvus Server?

### DISCUSSION THREAD

**Discussion by Pean-Mura** [milvus-io/milvus#5717] [Q&A and General discussion]
I found that if i know the server ip and port, I can directly get connected to the milvus server. And then I can upload or delete collection、partition or something like that from the server, how to avoid this?

**Comment by yhmo** [comment]
Milvus doesn't support authentication. There is an issue #254 about this, but we didn't implement it.
I heard some users create a middle-ware to use mongodb store authentication so that different user can access different milvus server. This work also can be done in front-end. Basic rule is hide the milvus server, build a middle-ware to isolate client.
